lib/api_auth/base.rb

```diff
@@ -67,6 +67,8 @@ def generate_secret_key
 
   private
 
+    AUTH_HEADER_PATTERN = /^APIAuth ([^:]+):(.+)$/
```

Comment:
Ah, I didn't catch that at all. I think it's legitimate to leave the caret off; I'll restore that and add a test explicitly covering it so we don't break it in the future.